project-reports/2023/2023-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,124 @@
+
+![alt_text](images/ffq3image1.png "image_tooltip")
+
+Contributors are active in the project and since June there were 14 new contributors who performed at least one commit.
+
+There are currently 748k lines of code for Hyperledger FireFly across 19 repositories, with a total of 7.7k commits to date.
+
+# Questions/Issues for the TOC
+
+Given the maturity of the project, and the recent changes in maintainership, FireFly meets the required and recommended criteria for Incubation Exit and will be submitting a separate proposal to the TOC to become a Graduated Project.
+
+# Releases
+
+## [FireFly v1.2.2](https://github.com/hyperledger/firefly/releases/tag/v1.2.2)
+
+FireFly v1.2.2 is a patch that fixes several issues, including:
+
+- FabConnect not being able to set transactiondex and eventindex
+- Correctly parse blockchain subscription names when delivering events
+
+## [FireFly v1.2.1](https://github.com/hyperledger/firefly/releases/tag/v1.2.1)
+
+FireFly v1.2.1 is a patch release that contains many fixes, including:
+
+- Fixed postgres optimizations, fail gracefully when inserting many data rows
+- Replace “UpsertBatch” with “InsertOrGetBatch”
+- Enable workflows for release branch
+- Backport fixes
+- FIxed multiple named tuple result from contract
+
+# Overall Activity in the Past Quarter
+
+The major highlights have been detailed at the beginning of the report. In summary, there has been a lot of activity in the project. New community members continue to discover the project and make contributions. This quarter has seen the most significant new contribution from outside the set of the original maintainers, in the form of an entirely new blockchain connector.
+
+# Current Plans
+
+## FireFly v1.3.0
+
+The next feature release of FireFly is currently in active development and is expected to be available in early Q4 2023. It will include many new exciting features:
+
+- Significantly enhanced onchain / offchain data coordination including the ability to attach a FireFly message to a custom contract invocation
+- Additional APIs for deleting FireFly Resources such as Token Pools, Contract Interfaces etc.
+- Configurable mTLS security across FireFly microservices for internal service-to-service authentication and encryption
+- PostgreSQL support for EVMConnect persistence
+- ERC-1155 token connector improvements for supporting a wider variety of ERC-1155 contracts
+- Performance improvements in transaction throughput and latency
+
+An additional, significant architectural roadmap item is active-active HA for FF FireFly Core. This will offer scaling improvements, increased resiliency, and more flexible deployment and migration options. This work is currently in the design phase and the related work is being tracked in GitHub.
+
+# Maintainer Diversity
+
+FireFly now has 12 maintainers representing three different companies:
+
+- [Kaleido](https://kaleido.io/)
+- [Fidelity](https://www.fidelity.com/)
+- [OneOf](https://www.oneof.com/)
```

Comment:
Is it correct to say that we have 10 maintainers from Kaleido, 1 maintainer from Fidelity, and 1 maintainer from OneOf?

Is the 1 maintainer from Instinctools the same as that from OneOf?

Reply:
> Is it correct to say that we have 10 maintainers from Kaleido, 1 maintainer from Fidelity, and 1 maintainer from OneOf?

Yes that is correct.

> Is the 1 maintainer from Instinctools the same as that from OneOf?

Yes, sorry I realize this wasn't very clear in the report. I will re-word it to make that more clear.

Reply:
If I read correctly OneOf/Instinctools is the same maintainer. Thanks for the clarification.